internal/codegen/tpl_stencil.go

```diff
@@ -45,14 +44,75 @@ type TplStencil struct {
 //	  {{ . }}
 //	{{- end }}
 func (s *TplStencil) GetModuleHook(name string) []interface{} {
-	k := path.Join(s.t.Module.Name, name)
-	v := s.s.sharedData[k]
+	k := s.s.sharedData.key(s.t.Module.Name, name)
+	v := s.s.sharedData.moduleHooks[k]
 
 	s.log.WithField("template", s.t.ImportPath()).WithField("path", k).
 		WithField("data", spew.Sdump(v)).Debug("getting module hook")
 	return v
 }
 
+// SetGlobal sets a global to be used in the context of the current template module
+// repository. This is useful because sometimes you want to define variables inside
+// of a helpers template file after doing manifest argument processing and then use
+// them within one or more template files to be rendered; however, go templates limit
+// the scope of symbols to the current template they are defined in, so this is not
+// possible without external tooling like this function.
+//
+// This template function stores (and its inverse, GetGlobal, retrieves) data that is
+// not strongly typed, so use this at your own risk and be averse to panics that could
+// occur if you're using the data it returns in the wrong way.
+//
+//	{{- /* This writes a global into the current context of the template module repository */}}
+//	{{ stencil.SetGlobal "IsGeorgeCool" true }}
```

Comment:
```suggestion
//	{{ stencil.SetGlobal "IsGeorgeCool" false }}
```

Reply:
lying in the codebase like that would be unethical jared